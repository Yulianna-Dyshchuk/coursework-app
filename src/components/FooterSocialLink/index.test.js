import React from "react";
import { shallow } from "enzyme";
import { FooterSocialLink } from "./index";

describe("FooterSocialLink component", () => {
  it("should render correctly without props", () => {
    const component = shallow(<FooterSocialLink />);

    expect(component).toMatchSnapshot();
  });

  it("should render correctly with props", () => {
    const component = shallow(<FooterSocialLink to="test" icon={null} />);

    expect(component).toMatchSnapshot();
  });
});
