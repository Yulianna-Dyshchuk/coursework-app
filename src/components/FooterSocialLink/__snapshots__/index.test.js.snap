// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterSocialLink component should render correctly with props 1`] = `
<a
  href="test"
/>
`;

exports[`FooterSocialLink component should render correctly without props 1`] = `
<a
  href=""
/>
`;
